forms/chapter-organizer-form.md

```diff
@@ -0,0 +1,70 @@
+# PyLadies Chapter Organizer Registration Form
+
+Please fill in this form if you are:
+
+- Currently a PyLadies chapter organizer
+- Was a chapter organizer and wants to step down
+- A new organizer for an existing chapter.
+
+Please share this form with your co-organizers, as well as please remember
+to fill this form again with any changes in your chapter (i.e. someone
+new is joining the team or someone is stepping down).
+
+We may request you to fill this form annually, to keep our records up to date.
+
+## How will the data here be used?
+
+PyLadies [Admin-and-logistics team members](https://github.com/pyladies/project-admin-logisitics#team)
+will refer to this data if we need to get in touch
+with local chapter organizers, for example for verifying the chapter's status
+or for verifying password reset request.
+
+Other PyLadies members and organizers may want to contact a chapter directly
+for various purposes, for example: sharing conference discount codes,
+sharing job opportunities, request speakers from your community, et.
+
+## Who has access to this form?
+
+The information collected here will be made available to all PyLadies chapter
+organizers, as a Google Spreadsheet. The Google Spreadsheet can be viewed
```

Comment:
Recommend s/Spreadsheet/Sheet (generic to specific name for clarity)

Reply:
@amcasari is this to minimize bias / product endorsement? We are actually using Google Docs since this allows us to tap into the Google security of our already existing Google `pyladies.com` account.